### ISSUE TITLE
Training Objectives and Scheduler Type

### ISSUE THREAD

**Issue by jibaixu** [Robert-gyj/Ctrl-World#10]
Hello, thank you very much for your work.

I noticed that the target of the unet in the training code(scripts/train_wm.py) and paper is to predict **x0**
but in the SVD scheduler(https://huggingface.co/stabilityai/stable-video-diffusion-img2vid/blob/main/scheduler/scheduler_config.json), the "prediction_type" is set to "**v_prediction**". Is there a problem with this?

**Comment by Robert-gyj** [comment]
Hi:
Thank you for the interests in our work!
Actually, SVD use an EDM version of velocity prediction, which makes the notation a little different from original v-pred. The unet output in https://github.com/Robert-gyj/Ctrl-World/blob/4c713d18e458a3ae70911572df2c4a8bf659b931/models/ctrl_world.py#L205 should be:

$$
v' = \frac{1}{\sqrt{1+\sigma^2}}\epsilon - \frac{\sigma}{\sqrt{1+\sigma^2}} x_0
$$

we just linear combine it in the end to reconstruct x_0 and calculate the loss.

Hope it helps!
